index.html

```diff
@@ -77,29 +83,38 @@ <h2 class="lets-connect">LET'S CONNECT</h2>
             alt="Project 1 Snapshot"
           />
           <div class="project-description-block">
-            <h2 class="card-title">Tonic</h2>
+            <h2 class="card-title mobile-display">Tonic</h2>
+            <h2 class="card-title big-size">Facebook 360</h2>
             <ul class="project-role-detail">
-              <li class="project-role-client">CANOPY</li>
+              <li class="project-role-client mobile-display">CANOPY</li>
+              <li class="project-role-client big-size">FACEBOOK</li>
```

Comment:
Well done on incorporating the changes suggested by the previous reviewer! I have a small suggestion that might further improve the appearance of your work cards and bring them closer to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-848&t=IpE91HjZFKnjsiLo-0). 🙋‍♂️

- [ ] **[OPTIONAL]** I suggest verifying the **font-weight** of the `client`, `role`, and `year` elements on your work cards. As per the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-848&t=IpE91HjZFKnjsiLo-0), these elements should have a `font-weight` of **400**, and the first element should have a `font-weight` of `500`. Keep up the excellent work! 💯

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236535710-fea2c75a-57ec-496e-9dda-6ca8a462bf68.png)

  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/236536127-c684ee71-8503-4402-9ad4-6aa8e6470d95.png)